lib/msal-common/src/cache/entities/CredentialEntity.ts

```diff
@@ -81,7 +81,7 @@ export class CredentialEntity {
             this.target,
             this.familyId,
             this.tokenType,
-            this.requestedClaimsHash
+            this.requestedClaimsHash,
```

Comment:
Did you mean to add the `userAssertionHash` to the key?

Reply:
No. Initially considered it, but since we are supporting only short term OBO, it is not necessary now. Will need it once we add RT flows.